sources/packages/web/src/types/contracts/EducationProgram.ts

```diff
@@ -50,12 +50,12 @@ export class EntranceRequirements {
 }
 
 export interface ProgramOfferingHeader {
-  institutionId: number;
-  institutionName: string;
-  submittedDate: string;
+  institutionId?: number;
+  institutionName?: string;
+  submittedDate?: Date;
   status: ProgramStatus | OfferingStatus;
-  assessedBy: string;
-  assessedDate: string;
+  assessedBy?: string;
+  assessedDate?: Date;
```

Comment:
Why are these optional? I see all these fields are used in both the vie files, Let me know if Iam missing anything